src/quic_transport.cpp

```diff
@@ -13,11 +13,13 @@ namespace quicr {
     std::shared_ptr<ITransport> ITransport::MakeClientTransport(const TransportRemote& server,
                                                                 const TransportConfig& tcfg,
                                                                 TransportDelegate& delegate,
+                                                                std::shared_ptr<TickService> tick_service,
                                                                 std::shared_ptr<spdlog::logger> logger)
     {
         switch (server.proto) {
             case TransportProtocol::kQuic:
-                return std::make_shared<PicoQuicTransport>(server, tcfg, delegate, false, std::move(logger));
+                return std::make_shared<PicoQuicTransport>(
+                  server, tcfg, delegate, false, tick_service, std::move(logger));
```

Comment:
```suggestion
                  server, tcfg, delegate, false, std::move(tick_service), std::move(logger));
```
To avoid the double copy.